xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
sure, i'll continue in the initial thread - the black sreen issue is solved (Winrenderer::m_iYV12RenderBuffer should be initialized with 1 instead 0 because the first picture after start / change is filled at position nextYVBuffer() and this is 1. Bt I still have some gap until the movie continues (0.2 sek)